### ISSUE TITLE
Installation with new ZendSkeletonApplication installs all components

### ISSUE THREAD

**Issue by RalfEggert** [bjyoungblood/BjyProfiler#61]
When I install BjyProfiler with the new ZendSkeletonApplication and Zend\Mvc 3 all ZF components are installed. Even Barcode and Captcha. I have no idea why this happens but it should be solved.

**Comment by andrey-mokhov** [comment]
for ZF3 can use:
```console
composer require andrey-mokhov/bjy-profiler:dev-master
```

**Comment by andrey-mokhov** [comment]
Released  version 1.3.0

[See changes](https://github.com/andrey-mokhov/bjy-profiler/releases/tag/1.3.0)

```console
$ composer require --dev andrey-mokhov/bjy-profiler:^1.3
```
